OneSignalSDK/unittest/src/test/java/com/test/onesignal/GenerateNotificationRunner.java

```diff
@@ -2303,6 +2304,59 @@ public void testNotificationWillShowInForegroundHandler_workTimeLongerThanTimeou
       assertNotificationDbRecords(1);
    }
 
+   /**
+    * Small icon accent color uses string.xml value in values-night when device in dark mode
+    */
+   @Test
+   @Config(qualifiers = "night")
+   public void shouldUseDarkIconAccentColorInDarkMode_hasMetaData() throws Exception {
+      OneSignal.initWithContext(blankActivity);
+      // Add the 'com.onesignal.NotificationAccentColor.DEFAULT' as 'FF0000AA' meta-data tag
+      OneSignalShadowPackageManager.addManifestMetaData("com.onesignal.NotificationAccentColor.DEFAULT", "FF0000AA");
+
+      BigInteger defaultColor = OneSignal_getAccentColor(new JSONObject());
+      assertEquals("FFFF0000", defaultColor.toString(16).toUpperCase()); // first param find from strings.xml instead of hardcode
```

Comment:
you might want to remove the comment  // first param find from strings.xml instead of hardcode

Reply:
Removed, and squashed commits